### PR TITLE
Fix typo and using of deprecated module

### PR DESCRIPTION
**Pull request by b0oh**

### CODE REVIEW

**Comment by whatyouhide** [comment]
👍

**Comment by whatyouhide** [comment]
Note that `Dict` has been **officially** deprecated in https://github.com/elixir-lang/elixir/commit/74825645e8cac770708f45139e651fd9d4e4264c. :)

**Comment by whatyouhide** [comment]
Ping @edgurgel? We (Football Addicts) have a plan of using this library in production and we'd love to help smooth a bunch of things (such as the ones in this PR) out, this is why I'm actively pinging :)

File: lib/gcm.ex [hunk 1]
```diff
@@ -15,7 +15,7 @@ defmodule GCM do
   """
   alias HTTPoison.Response
 
-  @base_url "https://gcm-http.googleapis.com/gcm"
+  @service_url "https://gcm-http.googleapis.com/gcm/send"
```
Comment (edgurgel) [inline lib/gcm.ex]: I would prefer to keep the URL as it is. Do you mind reverting this?

File: lib/gcm.ex [hunk 2]
```diff
@@ -64,7 +64,7 @@ defmodule GCM do
     { :error, :unauthorized }
   end
   defp build_response(_, %Response{ status_code: 503 }) do
-    { :error, :service_unavaiable }
+    { :error, :service_unavailable }
```
Comment (edgurgel) [inline lib/gcm.ex:67]: good catch XD

**Comment by edgurgel** [comment]
Awesome! I'm currently away but I will be back next week to release a new version with the proposed changes! Thank you very much @whatyouhide and @b0oh :)

Sorry for such late reply :(

**Comment by whatyouhide** [comment]
@edgurgel that sounds good, thanks for getting back to us :)

**Comment by whatyouhide** [comment]
Ping @edgurgel?

File: lib/gcm.ex (hunk 1 above)
Comment (b0oh) [inline lib/gcm.ex]: Done.

**Comment by edgurgel** [comment]
Sorry for such late replies! I'm back from holidays! 👍 I will be publishing a new version today!

**Comment by edgurgel** [comment]
I just published 1.4.0: https://hex.pm/packages/gcm/1.4.0

**Comment by whatyouhide** [comment]
Thanks @edgurgel! 💟

**Pull request merged**